File: src/classes/api/req_res/get-kmemos-response.ts
// ˅
'use strict';

import { Kmemo } from '@/classes/datas/kmemo';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class GetKmemosResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    kmemos: Array<Kmemo>;

    constructor() {
        // ˅
        super()
        this.kmemos = new Array<Kmemo>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/app/gkill/api/req_res/get_kmemos_response.go
// ˅
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

// ˄

type GetKmemosResponse struct {
	// ˅

	// ˄

	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	Kmemos []*reps.Kmemo `json:"kmemos"`

	// ˅

	// ˄
}

// ˅

// ˄
